gmachine.go

```diff
@@ -46,20 +48,20 @@ var TranslateTable = map[string]Instruction{
 	"DECA": {Opcode: DECA, Operands: 0},
 	"INCA": {Opcode: INCA, Operands: 0},
 	"BIOS": {Opcode: BIOS, Operands: 2},
+	"CMP":  {Opcode: CMP, Operands: 1},
+	"JMP":  {Opcode: JMP, Operands: 1},
 }
 
 type Word uint64
 
 type GMachine struct {
-	A, E, P        Word
+	A, P, T        Word
```

Comment:
The purpose of `T` isn't clear. What about calling it `Z` instead, for 'zero'?

Reply:
...and making it boolean, since the last `CMP` instruction either resulted in zero or not.